todo_app/tests/test_trello_items.py

```diff
@@ -0,0 +1,58 @@
+from todo_app.data.viewmodel import ViewModel
+from todo_app.data.Item import Item
+from todo_app.tests.dummy_data import get_dummy_cards_response
+
+
+def test_todo_items():
+
+    # Arrange
+    cards = []
+    trellolists = get_dummy_cards_response()
+    for trellolist in trellolists:
+        for card in trellolist ["cards"]:
+            item = Item.from_trello_card(card, trellolist)
```

Comment:
These are supposed to be unit tests, so it would be a bit nicer to skip over this stuff and just check the viewmodel's sorting.

i.e. create the list of `Item` objects directly instead of using Trello-like data.